content/chapter-destinations/clickhouse/_index.md

```diff
@@ -42,6 +42,8 @@ destination {
 };
 ```
 
+If you have a proto file, you can use the [`proto-var()`](#proto) option instead of `schema()` to create the message from a FilterX-formatted object.
```

Comment:
I would skip the "if you have a proto file" part from all places and mention that if the user has a formatted protobuf message (reference the `protobuf_message()` function) then they can use it here by specifying the name of the variable.

(We'd like to support FilterX profo schema description in the future, so a proto-file will not be mandatory in the future.)